src/Orleans.Core/Networking/ConnectionManager.cs

```diff
@@ -325,8 +323,9 @@ public async Task CloseAsync(SiloAddress endpoint)
                     }
                 }
 
-                await Task.WhenAll(closeTasks);
+                return Task.WhenAll(closeTasks);
```

Comment:
await is used here for debuggability - to retain stacks. I prefer that over performance for cold paths like this.
Was there a reason other than performance for this change?